app/src/main/java/ru/vssemikoz/newsfeed/di/DisposableModule.java

```diff
@@ -0,0 +1,13 @@
+package ru.vssemikoz.newsfeed.di;
+
+import dagger.Module;
+import dagger.Provides;
+import io.reactivex.rxjava3.disposables.CompositeDisposable;
+
+@Module
+public class DisposableModule {
```

Comment:
This is not needed. We usually create an Rx chain in a single place or in concrete classes like presenters. You can utilize `BasePresenter` (or your single presenter in your case) to have a `disposable` object to accumulate your disposables (subscriptions) there. So, each child's class will be able to access and use it.